app/src/main/java/com/keylesspalace/tusky/components/account/AccountViewModel.kt

```diff
@@ -85,7 +85,7 @@ class AccountViewModel @Inject constructor(
                 mastodonApi.relationships(listOf(accountId))
                     .fold(
                         { relationships ->
-                            relationshipData.postValue(Success(relationships[0]))
+                            relationshipData.postValue(if (relationships.isNotEmpty()) Success(relationships[0]) else Error())
```

Comment:
It was similar to this in 2020 but the size check was then removed by a sort-of refactoring.